### PR TITLE
Update README.md links to some common commands

### PR DESCRIPTION
**Pull request by amanex007**
Added CLI commands here in response to [issue 179](https://github.com/Cloud-CV/evalai-cli/issues/179)

IMPORTANT NOTES (please read, then delete):

The PR title should start with "Fix #bugnum: " (if applicable), followed by a clear one-line present-tense summary of the changes introduced in the PR. For example: "Fix #bugnum: Introduce the first version of the collection editor.".

Please make sure to mention "#bugnum" somewhere in the description of the PR. This enables Github to link the PR to the corresponding bug.

Please also make sure to follow the [style rules](https://github.com/Cloud-CV/evalai-cli/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by pushkalkatara** [comment]
Hi @amanex007 Thanks for the PR. Overall it looks good to me.
I have some minor suggestions - 
1. Please edit the PR description by removing the IMPORTANT NOTES (as written in the bracket - read and delete).
2. Some of the commands won't work as the documentation on the website is deprecated. There is an open issue to update that too - #174 . 
Replace
```
evalai challenge 1 participant 2
```
with 
```
evalai challenge 1 participate 2
```
and replace
```
evalai challenges future
```
Rest everything is fine.

**Comment by yashdusing** [comment]
@amanex007 it would be nice to have the screenshots as well along with the description.

File: README.md [hunk 1]
```diff
@@ -52,84 +52,83 @@ ________________________________________________________________________________
...
-    ```bash
+    ```$
         evalai challenges ongoing
     ```
     
-Display the upcoming challenges                                 
-    ```bash
+Display the future challenges                                 
+    ```$
         evalai challenges upcoming
```
Comment (pushkalkatara) [inline README.md]: please change upcoming -> future

File: README.md [hunk 2]
```diff
@@ -47,3 +47,136 @@ Two users will be created by default which are listed below -
...
+
+Display the details of the phase with the given phase ID (4) of the challenge with the given challenge ID (1)                                
+    ```$
+        evalai challenge 1 phase 4
+    ```
+
+
+## Teams    
+
+ Participate in a challenge with the given challenge ID (1) and given team ID (2)
```
Comment (Ram81) [inline README.md]: @amanex007 can you make sure lines are consistent. Some lines start with a leading space but some dont. Can you please fix that

File: README.md [hunk 3]
```diff
@@ -47,3 +47,136 @@ Two users will be created by default which are listed below -
...
+
+## Teams    
+
+ Participate in a challenge with the given challenge ID (1) and given team ID (2)                                
+    ```$
+       evalai challenge 1 participate 2 
+    ``` 
+ 
+
+ View your participant teams
```
Comment (Ram81) [inline README.md]: Same here

**Pull request closed without merging**